docs/release-notes/cloud/cloud-2022-04-04-eslint-8-update.md

```diff
@@ -0,0 +1,59 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+---
+
+# Update of ESLint to version 8 with breaking changes April 4, 2022
+
+On April 4, 2022 we'll be updating the tool ESLint to version 8.10.0.
+
+ESLint 8 [introduces breaking changes](https://github.com/eslint/eslint/releases/tag/v8.0.0){: target="_blank"} that may affect your repositories, especially if you use a configuration file.
```

Comment:
If I understood things correctly during the conversation we had this morning, the breaking changes only affect customers that are using the ESLint configuration file.

Do you confirm this, @lolgab? If this is true, I'll make it more explicit that only customers with configuration files need to worry about the breaking changes. :+1:

Reply:
We don't have 100% proof that the UI config will be unaffected, but for sure we will make the proper changes to the generated configuration when users use UI config mode.
In case of configuration file it is on the users to make the changes, yes.

Reply:
I made an edit to clarify that only customers who are using a configuration file need to take action.